.github/workflows/ui-tests/cypress/integration/deposit_spec.js

```diff
@@ -0,0 +1,192 @@
+    })
+
+    it('sees expected external accounts', function () {
+        cy.get('#depositSpan').click()
+        cy.get('#depositFunds').should('be.visible')
+        cy.get('#accounts').children().first().as('firstOption')
+        cy.get('@firstOption').contains('External Bank')
+        cy.get('@firstOption').contains('9099791699')
+        cy.get('@firstOption').contains('808889588')
+    })
+
+    it('can deposit funds successfully', function () {
+        const depositAmount = validPayment()
+
+        cy.deposit(externalAccount, depositAmount)
+        cy.get('.alert').contains(depositMsgs.success)
+    })
+
+    it('can see balance update after deposit', function () {
+        const depositAmount = validPayment()
+        let expectedBalance
+        cy.get("#current-balance").then(($span) => {
+            const currentBalanceSpan = $span.text()
+            // regex: removes any characters that are not a digit [0-9] or a period [.]
+            const currentBalance = parseFloat(currentBalanceSpan.replace(/[^\d.]/g, ''))
+
+            expectedBalance = formatter.format(currentBalance + parseFloat(depositAmount))
+
+            cy.deposit(externalAccount, depositAmount)
+            cy.get('.alert').contains(depositMsgs.success)
+        })
+        cy.visit('/home')
+        cy.get('#current-balance').then(($span) => {
+            const updatedBalanceSpan = $span.text()
+            cy.wrap(updatedBalanceSpan).should('contain', expectedBalance)
+        })
+
+    })
+
+    it('can see transaction in history after deposit', function () {
+        const depositAmount = validPayment()
+
+        cy.deposit(externalAccount, depositAmount)
+        cy.get('.alert').contains(depositMsgs.success)
+
+        cy.visit('/home')
+
+        cy.get('#transaction-table').find('tbody>tr').as('latest')
+
+        cy.get('@latest').find('.transaction-account').contains(externalAccount.accountNum)
+        cy.get('@latest').find('.transaction-type').contains('Debit')
+        cy.get('@latest').find('.transaction-amount').contains(depositAmount)
+    })
+
+    it('can deposit to new account and see new account', function () {
+        const accountNum = validAccountNum()
+        const routingNum = validRoutingNum()
+        const newExternalAccount = {
+            accountNum: accountNum,
+            routingNum: routingNum,
+            contactLabel: `testcontact${accountNum}`
+        }
+        const paymentAmount = validPayment()
+
+        cy.depositToNewAccount(newExternalAccount, paymentAmount)
+        cy.get('.alert').contains(depositMsgs.success)
+
+        cy.reload()
+        cy.get('#depositSpan').click()
+        cy.get('#accounts').contains(newExternalAccount.contactLabel)
+        cy.get('#accounts').contains(newExternalAccount.accountNum)
+        cy.get('#accounts').contains(newExternalAccount.routingNum)
+    })
+
+})
+
+describe('Deposit is unsuccessful with invalid data', function () {
```

Comment:
maybe we should add a test deposit using the local routing number? (I'm actually not sure how that's handled internally. It seems to trigger the frontend bug we already know about)

Reply:
what is the default user's local routing number?

Reply:
nvm found it https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/kubernetes-manifests/config.yaml